session/pingpong/invoice_payer.go

```diff
@@ -250,7 +249,6 @@ func (ip *InvoicePayer) issueExchangeMessage(invoice crypto.Invoice) error {
 
 	ip.deps.EventBus.Publish(event.AppTopicInvoicePaid, event.AppEventInvoicePaid{
 		ConsumerID: ip.deps.Identity,
-		SessionID:  ip.deps.SessionID,
```

Comment:
Why remove this?

Reply:
Returned it back. Setting now SessionID after session established.